crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -98,7 +98,8 @@ impl Object {
             .stored
             .owner_type
             .try_into()
-            .map_err(|e: IndexerError| graphql_error(code::INTERNAL_SERVER_ERROR, e.to_string()))?;
+            .map_err(|e: IndexerError| Error::Internal(e.to_string()))
```

Comment:
curious if we need to type e: IndexerError here?

Reply:
Good question, let's see!

Reply:
Sadly, yes -- probably because it can't infer through the associated type on the `TryInto` impl.